ansible_collections/arista/cvp/molecule/cv_image_v3/test_cv_image_v3.yaml

```diff
@@ -0,0 +1,164 @@
+---
+- name: Test cv_image_v3
+  hosts: CloudVision
+  connection: local
+  gather_facts: no
+  vars:
+    # Make sure given image does not exist on CVP
+    image_name: "EOS-4.25.4M.swi"
+  tasks:
+
+  ### Run all the tasks sequentially ###
+
+    - name: "Create bundle {{inventory_hostname}}"
```

Comment:
this task was failing to me as the image was not on the CV dut I was using and the comment above was saying it shouldn't exist so I had to create a few other tasks to upload the images first

e.g.:

```yaml
---
- name: Test cv_image_v3
  hosts: CloudVision
  connection: local
  gather_facts: no
  vars:
    # Make sure given image does not exist on CVP
    image_name: "vEOS-lab-4.25.5M.swi"
    extension_name: "TerminAttr-1.25.1-1.swix"
    extension_name_2: "TerminAttr-1.26.0-1.swix"
    ansible_command_timeout: 1200
    ansible_connect_timeout: 600
  tasks:

  ### Run all the tasks sequentially ###
    - name: "Upload EOS image to {{inventory_hostname}}"
      arista.cvp.cv_image_v3:
        mode: image
        action: add
        image: "vEOS-lab-4.25.5M.swi"
    - name: "Upload TerminAttr image to {{inventory_hostname}}"
      arista.cvp.cv_image_v3:
        mode: image
        action: add
        image: "TerminAttr-1.25.1-1.swix"
    - name: "Upload Second TerminAttr image to {{inventory_hostname}}"
      arista.cvp.cv_image_v3:
        mode: image
        action: add
        image: "TerminAttr-1.26.0-1.swix"  
```

Reply:
Fixed!